webpack/components/meetup.js

```diff
@@ -16,13 +16,18 @@ export function getNextMeetup(callback) {
 
 export function renderNextMeetup(meetupData) {
   const nextMeetup = meetupData[0];
-  const nextMeetupDate = new Date(nextMeetup.local_date+"T"+nextMeetup.local_time);
+  const nextMeetupDate = new Date(nextMeetup.time);
   const dateOutputOptions = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric'};
   const dateOutput = nextMeetupDate.toLocaleString("en-US", dateOutputOptions);
+  const location = ["name", "address_1", "city"].map(function(p) {
+    return nextMeetup.venue[p]
+  }).filter(function(addressPart) {
+    return addressPart;
+  }).join(", ");
 
   document.getElementById("meetup-event-title").innerHTML = nextMeetup.name;
   document.getElementById("meetup-event-datetime").innerHTML = dateOutput;
-  document.getElementById("meetup-event-location").innerHTML = nextMeetup.venue.name+", "+nextMeetup.venue.address_1+", "+nextMeetup.venue.city;
```

Comment:
`address_1` was showing as `undefined` for our next meetup
![Screen Shot 2019-05-22 at 19 04 09](https://user-images.githubusercontent.com/580014/58161867-69aeac00-7cc4-11e9-9a06-0aa9d05ebe3e.jpg)